microscopic-monks/primal/screens/game_screen.py

```diff
@@ -0,0 +1,260 @@
+import random
+
+from primal.engine.screen import Screen
+from primal.engine.camera import OrthographicCamera
+from primal.engine.sprite import Player, ColorSprite
+from primal.engine.world import World
+from primal.engine import keys
+from primal.screens.death_screen import DeathScreen
+
+from primal.gui.health import HealthBar
+from primal.gui.inventory import Inventory
+
+
+class GameScreen(Screen):
+    VP_WIDTH = 1280
+    VP_HEIGHT = 720
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.remove = 1
+
+        self.dividend = 10
+
+        self.clicked_features = dict()
+        self.last_clicked = 0.0
+
+        self.camera = OrthographicCamera(self.canvas, self.VP_WIDTH, self.VP_HEIGHT)
+        self.camera.start_region()
+
+        self.zoom = 1.0
+
+        self.world = World((0, 0))
+        self.world.draw(self.canvas)
+
+        self.player = Player('player.png', (0, 0), (40, 80), 0)
```

Comment:
arguments should be self-referencing, it's hard to know what all the arguments mean.